server/dive_server/crud_rpc.py

```diff
@@ -244,6 +244,20 @@ def run_training(
 
     # Ensure the folder to upload results to exists
     results_folder = training_output_folder(user)
+    output_folders = list(
+        Folder().find(
+            query={
+                'parentId': results_folder['_id'],
+                'name': pipelineName,
```

Comment:
`pipelineName` needs to have spaces replaced with '_'.  It seems to do that in the output so `test folder` as the train name becomes a folder called `test_folder` and we need to check against that instead of the direct name.

Reply:
Great catch.  I removed the underscore replacement line because there's no reason folders have to have names without spaces.  Space is a pretty arbitrary character to disallow considering that this is a totally freeform input string.

Reply:
it may get a bit weirder with the fact that `'test'` and `'test    '` may look identical in the trained pipeline list but I guess that is more the fault of the user.

Reply:
`strip()` is part of the girder-side preprocessing for entity names.  If you ask for `'test '` it gets trimmed and then collides.